scripts/smokeTests.sh

```diff
@@ -27,60 +30,63 @@ if (( CURRENT_NODE_VERSION < REQUIRED_NODE_VERSION )); then
 fi
 
 # Autodetect project directory relative to this script's path
-PROJECT_DIR="$0"
-while [ -h "$PROJECT_DIR" ]; do
-    ls=$(ls -ld "$PROJECT_DIR")
-    link=$(expr "$ls" : '.*-> \(.*\)$')
-    if expr "$link" : '/.*' > /dev/null; then
-        PROJECT_DIR="$link"
-    else
-        PROJECT_DIR="$(dirname "$PROJECT_DIR")/$link"
-    fi
-done
-PROJECT_DIR="$(dirname "$PROJECT_DIR")/.."
-PROJECT_DIR="$(cd "$PROJECT_DIR"; pwd)"
+PROJECT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Comment:
The above code can deal with symbolic links, I've been using this template for years in my scripts.